### PR TITLE
v1alpha2 TFEvent metrics collector

### PR DESCRIPTION
**Pull request by richardsliu**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/538)

### CODE REVIEW

File: scripts/v1alpha2/build.sh [hunk 1]
```diff
@@ -29,4 +29,4 @@ echo "Building core image..."
 docker build -t ${PREFIX}/v1alpha2/katib-controller -f ${CMD_PREFIX}/katib-controller/v1alpha2/Dockerfile .
 docker build -t ${PREFIX}/v1alpha2/katib-manager -f ${CMD_PREFIX}/manager/v1alpha2/Dockerfile .
 docker build -t ${PREFIX}/v1alpha2/metrics-collector -f ${CMD_PREFIX}/metricscollector/v1alpha2/Dockerfile .
-
+docker build -t ${PREFIX}/v1alpha2/tfevent-metrics-collector -f ${CMD_PREFIX}/tfevent-metricscollector/v1alpha2/Dockerfile .
```
Comment (hougangliu) [inline scripts/v1alpha2/build.sh]: please also enable image push in prow post/pre-submit
gcloud builds submit . --tag=${REGISTRY}/${REPO_NAME}/v1alpha2/metrics-collector:${VERSION} --project=${PROJECT}
gcloud builds submit . --tag=${REGISTRY}/${REPO_NAME}/v1alpha2/tfevent-metrics-collector:${VERSION} --project=${PROJECT}

Thanks!

Comment (richardsliu) [reply scripts/v1alpha2/build.sh]: Done, thanks!

**Pull request merged**